app/models/landing_page/block/statistics.rb

```diff
@@ -30,15 +29,15 @@ def rows
       rows
     end
 
+    def attachment
+      @attachment ||= landing_page.attachments.find { |att| att.title == data["attachment_csv"] }
+    end
+
   private
 
     def csv_rows
-      rows = CSV.read(csv_file_path, headers: true).map(&:to_h)
-      rows.each(&:deep_symbolize_keys!)
-    end
-
-    def csv_file_path
-      @csv_file_path ||= Rails.root.join("#{STATISTICS_DATA_PATH}/#{data['csv_file']}")
+      rows = CSV.new(URI.parse(attachment.url).open, headers: true).map(&:to_h)
```

Comment:
Could we also allow the CSVs to be read in from an csv file in whilst we're relying on local demos for testing?

I'm concerned that until one of these pages go live, it will be super difficult to make frontend changes to the statistics block.

Reply:
I think there's a longer conversation to be had about this, for the moment I've reinstated scaffolding code.